src/settings/CentralOrdering/CentralOrdering.test.js

```diff
@@ -33,6 +33,16 @@ const mockKy = {
 };
 const mockData = { id: 'setting-id' };
 
+const handleConfirmModal = (confirm = true) => {
+  const name = confirm ? 'ui-consortia-settings.button.confirm' : 'stripes-components.cancel';
```

Comment:
can we use regExp here ? like /confirm/ : /cancel/

Reply:
sure, updated